client/me/account/index.jsx

```diff
@@ -202,8 +198,16 @@ module.exports = React.createClass( {
 	renderHolidaySnow() {
 		// Note that years and months below are zero indexed
 		let today = this.moment(),
-			startDate = this.moment( { year: today.year(), month: 11, day: 1 } ),
-			endDate = this.moment( { year: today.year(), month: 0, day: 4 } );
+			startDate = this.moment( {
+				year: today.year(),
+				month: 11,
+				day: 1
+			} ),
+			endDate = this.moment( {
+				year: today.year(),
+				month: 0,
+				day: 4
+			} );
```

Comment:
I think every change from line `210` and above in this file is due to your auto-formatter. I don't have an issue with that, but could we break it into a separate commit that way the change needed for the issue is more clear?